RunningOrder/Story/Models/StoryInformation.swift

```diff
@@ -35,10 +35,9 @@ struct StoryInformation {
 
     private var videoExtension: String?
 
-    init(storyId: Story.ID, configuration: Configuration = Configuration(), links: [Link] = [], steps: [String] = [], videoUrl: URL? = nil) {
+    init(storyId: Story.ID, configuration: Configuration = Configuration(), steps: [String] = [], videoUrl: URL? = nil) {
```

Comment:
could you please revert this to enable the possibility to add links at init ?

Reply:
Already resolved